pyjetty/alice_analysis/analysis/user/substructure/run_analysis.py

```diff
@@ -174,75 +180,79 @@ def create_output_subdir(self, output_dir, name):
   # Main processing function
   #---------------------------------------------------------------
   def run_analysis(self):
-    
+
     if self.do_unfolding:
       self.perform_unfolding()
-    
+
     # Loop through jet radii
     for jetR in self.jetR_list:
-    
+
       # Loop through subconfigurations to unfold
       for i, _ in enumerate(self.obs_subconfig_list):
 
         obs_setting = self.obs_settings[i]
         grooming_setting = self.grooming_settings[i]
         obs_label = self.utils.obs_label(obs_setting, grooming_setting)
-        reg_param_final = self.utils.get_reg_param(self.obs_settings, self.grooming_settings, self.obs_subconfig_list, self.obs_config_dict, obs_label, jetR)
```

Comment:
I think we should still keep this functionality -- to be able to manually override the automatically determined regularization parameter. Can you keep this in, such that if we specify a manual value in the config it uses that?

Reply:
Resolved in `c9433d0`